src/services/collector.ts

```diff
@@ -7,7 +7,7 @@ import dateHelper from "../utils/dateHelper";
 import fs from 'fs';
 import path from 'path';
 import {Pricer} from "./pricer";
-import {ignored} from "../index"
```

Comment:
@josh-burns I did the little refacto I tried to explain to you. You don't need to import the ignored var as you already pass it to the constructor of the collector class.